ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh

```diff
@@ -57,6 +53,15 @@ else
 
 fi
 
+#to support vsphere-ipi-zones-multisubnets-external-lb-f28 and vsphere-ipi-zones-multisubnets-external-lb-f28-destructive.add these var to specify the portgroups.
```

Comment:
```suggestion
#to support vsphere-ipi-zones-multisubnets-external-lb, which need 3 different subnets, add these var to specify the portgroups.
```

Reply:
@WenXinWei do not forget this minor comment

Reply:
@WenXinWei once this is resolved, this PR should be ready to merge, and do not need more rehearse testing any more.